shadows/framework/src/main/java/org/robolectric/shadows/ShadowBluetoothDevice.java

```diff
@@ -109,6 +109,21 @@ public void setName(String name) {
     this.name = name;
   }
 
+  /**
+   * Set the alias for bluetooth device.
+   *
+   * @param alias The alias name that set to bluetooth device.
+   * @return If API is larger than or equals to S, it returns [BluetoothStatusCodes] code, otherwise
+   *     it returns boolean.
+   */
+  public Object setAlias(String alias) {
```

Comment:
do we want to change the parameter type to Object to match the original function signature before my MR happened? do we also want to add @Deprecate annotation to advocate developers to call the function on real object?